app/matplotlib.ipynb

```diff
@@ -233,22 +253,42 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
    "source": [
     "x = np.linspace(0, 5, 100)\n",
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
+    "fig, axs = plt.subplots(1, 2, figsize=(10, 5))\n",
+    "\n",
+    "axs[0].plot(x, y1, color=\"blue\")\n",
+    "axs[0].set_title(\"Sin Function\")\n",
+    "axs[0].set_xlabel(\"x\")\n",
+    "axs[0].set_ylabel(\"sin(x)\")\n",
+    "\n",
+    "axs[1].plot(x, y2, color=\"red\")\n",
+    "axs[1].set_title(\"Cos Function\")\n",
+    "axs[1].set_xlabel(\"x\")\n",
+    "axs[1].set_ylabel(\"cos(x)\")\n",
+    "\n",
+    "fig.suptitle(\"Sin and Cos Functions\")\n",
     "\n",
-    "# write your code here\n"
+    "plt.tight_layout()\n",
+    "data = np.random.normal(0, 1, 1000)\n",
+    "plt.hist(data, bins=100)\n",
+    "plt.title(\"Random digits\")\n",
+    "\n",
+    "plt.xlabel(\"x\")\n",
+    "plt.ylabel(\"y\")\n",
```

Comment:
This histogram plot seems unrelated to Task 6, which is about plotting sin and cos functions in subplots. Consider removing these lines to keep the task focused.